src/libs/PersonalDetailsUtils.js

```diff
@@ -63,10 +63,22 @@ function getAccountIDsByLogins(logins) {
     return _.reduce(
         logins,
         (foundAccountIDs, login) => {
-            const currentDetail = _.find(personalDetails, (detail) => detail.login === login);
+            let isUserWithNoLogin = false;
+            const currentDetail = _.find(personalDetails, (detail) => {
+                if (!_.isUndefined(detail.login)) {
+                    isUserWithNoLogin = true;
+                }
+                return detail.login === login;
+            });
```

Comment:
I'm a little confused by this
- What is `isUserWithNoLogin` supposed to denote? we set it to false, and when there a `detail.login` values that is **not** undefined, we set it to true. That sounds backwards
- We're iterating over a personalDetails array that contains personal details for more than one user, yet we could be setting `isUserWithNoLogin` to true, even when we don't find a matching entry in the array where `detail.login === login`, is that right?